lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -0,0 +1,119 @@
+      description: 'Screenshot Thumbnails',
+      helpText: 'This is what the load of your site looked like.',
+      requiredArtifacts: ['traces']
+    };
+  }
+
+  /**
+   * Scales down an image to THUMBNAIL_WIDTH using nearest neighbor for speed, maintains aspect
+   * ratio of the original thumbnail.
+   *
+   * @param {{width: number, height: number, data: !Array<number>}} imageData
+   * @return {{width: number, height: number, data: !Array<number>}}
+   */
+  static scaleImageToThumbnail(imageData) {
+    const scaledWidth = THUMBNAIL_WIDTH;
+    const scaleFactor = imageData.width / scaledWidth;
+    const scaledHeight = Math.floor(imageData.height / scaleFactor);
+
+    const outPixels = new Uint8Array(scaledWidth * scaledHeight * 4);
+
+    for (let i = 0; i < scaledWidth; i++) {
+      for (let j = 0; j < scaledHeight; j++) {
+        const origX = Math.floor(i * scaleFactor);
+        const origY = Math.floor(j * scaleFactor);
+
+        const origPos = (origY * imageData.width + origX) * 4;
+        const outPos = (j * scaledWidth + i) * 4;
+
+        outPixels[outPos] = imageData.data[origPos];
+        outPixels[outPos + 1] = imageData.data[origPos + 1];
+        outPixels[outPos + 2] = imageData.data[origPos + 2];
+        outPixels[outPos + 3] = imageData.data[origPos + 3];
+      }
+    }
+
+    return {
+      width: scaledWidth,
+      height: scaledHeight,
+      data: outPixels,
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    const trace = artifacts.traces[Audit.DEFAULT_PASS];
+    const cachedThumbnails = new Map();
+
+    return artifacts.requestSpeedline(trace).then(speedline => {
+      const thumbnails = [];
+      const analyzedFrames = speedline.frames.filter(frame => !frame.isProgressInterpolated());
+
+      for (let i = 1; i <= NUMBER_OF_THUMBNAILS; i++) {
+        const targetTimestamp = speedline.beginning + speedline.complete * i / NUMBER_OF_THUMBNAILS;
+
+        let frameForTimestamp = null;
+        if (i === NUMBER_OF_THUMBNAILS) {
+          frameForTimestamp = analyzedFrames[analyzedFrames.length - 1];
+        } else {
+          analyzedFrames.forEach(frame => {
+            if (frame.getTimeStamp() <= targetTimestamp) {
+              frameForTimestamp = frame;
+            }
+          });
+        }
+
+        const imageData = frameForTimestamp.getParsedImage();
+        const thumbnailImageData = ScreenshotThumbnails.scaleImageToThumbnail(imageData);
+        const base64Data = cachedThumbnails.get(frameForTimestamp) ||
+            jpeg.encode(thumbnailImageData, 90).data.toString('base64');
+
+        cachedThumbnails.set(frameForTimestamp, base64Data);
+        thumbnails.push({
+          timing: Math.round(targetTimestamp - speedline.beginning),
+          timestamp: targetTimestamp * 1000,
+          data: base64Data,
+        });
+      }
+
+      return {rawValue: thumbnails};
```

Comment:
I think we assume that `rawValue` will be either a `boolean` or a `number` (which matters when creating the `score` from the `rawValue`). Not an issue here because the score is never used, but it would be nice to avoid having a special audit like this which only works because of the specific renderer path it hits. Ideally it would work if someone else wanted to mix it into their config without a perf section, for instance.

Can we mark it informative, have a `false` `rawValue` and stick the thumbnails in `details`?